pkg/volume/validation/pv_validation.go

```diff
@@ -56,3 +60,16 @@ func checkMountOption(pv *api.PersistentVolume) field.ErrorList {
 	}
 	return allErrs
 }
+
+// ValidateHostPath will make sure targetPath:
+// 1. does not have any element which is ".."
+func ValidateHostPath(targetPath string) error {
```

Comment:
There's tons of things this could check that it doesn't (max length, invalid chars, double slashes, etc, and naming it genetically encourages adding those types of checks which could break some callers' assumptions